Fixtures/Miscellaneous/ExeTest/Package.swift

```diff
@@ -1,10 +1,10 @@
-// swift-tools-version:5.3
+// swift-tools-version: 999.0
```

Comment:
This is what allows a test target to import and link against an executable target.